simplyblock_core/cluster_ops.py

```diff
@@ -202,12 +202,28 @@ def _set_max_result_window(cluster_ip, max_window=100000):
         logger.error(f"Failed to create template for future indices: {response_template.text}")
         return False
 
+
+def parse_protocols(input_str: str):
+    valid = {"tcp", "rdma"}
+
+    # split by comma, strip whitespace, and lowercase
+    parts = {p.strip().lower() for p in input_str.split(",")}
+
+    # validate input
+    if not parts.issubset(valid):
+        raise ValueError(f"Invalid protocol(s): {parts - valid}")
+
+    return {
+        "tcp": "tcp" in parts,
+        "rdma": "rdma" in parts,
+    }
    
 def create_cluster(blk_size, page_size_in_blocks, cli_pass,
                    cap_warn, cap_crit, prov_cap_warn, prov_cap_crit, ifname, mgmt_ip, log_del_interval, metrics_retention_period,
                    contact_point, grafana_endpoint, distr_ndcs, distr_npcs, distr_bs, distr_chunk_bs, ha_type, mode,
                    enable_node_affinity, qpair_count, max_queue_size, inflight_io_threshold, enable_qos, disable_monitoring, strict_node_anti_affinity, name, 
-                   tls_secret, ingress_host_source, dns_name, is_single_node) -> str:
+                   tls_secret, ingress_host_source, dns_name, fabric, is_single_node) -> str:
+                   tls_secret, ingress_host_source, dns_name) -> str:
```

Comment:
remove this line
`                   tls_secret, ingress_host_source, dns_name) -> str:`